packages/core/streams/openai-stream.test.tsx

```diff
@@ -39,34 +76,18 @@ describe('OpenAIStream', () => {
   });
 
   it('should be able to parse SSE and receive the streamed response', async () => {
-    const stream = OpenAIStream(
-      await fetch(server.api, {
-        headers: {
-          'x-mock-service': 'openai',
-          'x-mock-type': 'chat',
-        },
-      }),
-    );
+    const stream = OpenAIStream(await fetch(DEFAULT_TEST_URL));
     const response = new StreamingTextResponse(stream);
-    const client = createClient(response);
-    const chunks = await client.readAll();
+
+    const chunks = await readAllChunks(response);
+
     expect(JSON.stringify(chunks)).toMatchInlineSnapshot(
       `"[\\"Hello\\",\\",\\",\\" world\\",\\".\\"]"`,
     );
-    expect(JSON.stringify(server.getRecentFlushed())).toMatchInlineSnapshot(
-      `"[{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"role\\":\\"assistant\\"},\\"index\\":0,\\"finish_reason\\":null}]},{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"content\\":\\"Hello\\"},\\"index\\":0,\\"finish_reason\\":null}]},{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"content\\":\\",\\"},\\"index\\":0,\\"finish_reason\\":null}]},{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"content\\":\\" world\\"},\\"index\\":0,\\"finish_reason\\":null}]},{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"content\\":\\".\\"},\\"index\\":0,\\"finish_reason\\":null}]},{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{},\\"index\\":0,\\"finish_reason\\":\\"stop\\"}]}]"`,
```

Comment:
How important is the flush testing here? Does the first expect cover all that's needed or does this change remove coverage?

Reply:
I don't see any reason for it to be necessary